packages/core/src/callbacks/CallbackManager.ts

```diff
@@ -205,9 +212,10 @@ export class CallbackManager implements CallbackManagerMethods {
     if (!handlers) {
       return;
     }
+    const clone = structuredClone(detail);
```

Comment:
can we use structuredClone now? i remember you shouldn't because of node 16 compatibility?

Reply:
I'm not sure for now; I think we could use new APIs for the new module. And we could drop node16 support in 0.3.x

Reply:
/cc @yisding